backend/app/services/git_skill/utils.py

```
@@ -0,0 +1,535 @@
+        return ParsedRepoUrl(
+            provider=provider,
+            owner=owner,
+            repo=repo,
+            domain=host,
+            auth_info=auth_info,
+            base_url=base_url,
+        )
+
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)